lib/platform.py

```diff
@@ -178,14 +178,39 @@ def setup(self, client_path, fail_exit=False):
             uuids.pop()
             if not server.wait_for_generation("/share/".join(uuids), date0) and fail_exit:
               exit(1)
-  
+
           # update promises on the whole chain
           relays.append(server)
           relays.reverse()
           for h in relays:
             h.run("rudder agent run -u", quiet=False, live_output=True, fail_exit=fail_exit)
           host.run("rudder agent run -ui", quiet=False, live_output=True, fail_exit=fail_exit)
 
+  def pf_id_to_network(self, pf_id):
+    # Since the mask is always 255.255.255.0 this is reasonable
+    return ("192.168.%s.0"%(pf_id), "255.255.255.0")
```

Comment:
the base network can be defines in the Vagrantfile, so you shouldn't hardcode 192.168. here an the mask is also in the Vagrantfile (see $NETWORK)